server/main.c

```diff
@@ -1170,6 +1175,37 @@ np2srv_init_schemas(void)
     return -1;
 }
 
+#ifdef NP2SRV_ENABLED_URL_CAPABILITY
+static int
+np2srv_init_urlctx()
+{
+    /*
+     * Create a context for working with config XML in a standalone file.
+     *
+     * In standalone config files, the <config> node with namespace
+     * "urn:ietf:params:xml:ns:netconf:base:1.0" wraps xml data. We create a
+     * special schema module to define this to libyang so we can
+     * import/export these files. The overloading of the namespace prevents
+     * us from storing this module in our server context, so it must be kept
+     * in a new context.
+     */
```

Comment:
This was the correct solution currently. However, I have added this `config` virtual node into `ietf-netconf` so you should have no problems using the existing context. Please, simplify your code this way.